tardis/plasma/tests/test_partition_function.py

```diff
@@ -1,19 +1,80 @@
+import os
+import pytest
+import  yaml
+import h5py
 import numpy as np
 
-def test_level_boltzmann_factor_lte(level_boltzmann_factor_lte, levels):
-    assert np.allclose(level_boltzmann_factor_lte.ix[2].ix[0].ix[0], 1)
-    assert np.allclose(level_boltzmann_factor_lte.ix[2].ix[1].ix[0], 2)
-    assert np.allclose(level_boltzmann_factor_lte.ix[2].ix[0].ix[10],
-        7.6218092841240705e-12)
-
-def test_level_boltzmann_factor_dilute_lte(level_boltzmann_factor_dilute_lte,
-    levels):
-    assert np.allclose(level_boltzmann_factor_dilute_lte.ix[2].ix[0].ix[0], 1)
-    assert np.allclose(level_boltzmann_factor_dilute_lte.ix[2].ix[1].ix[0], 2)
-    assert np.allclose(level_boltzmann_factor_dilute_lte.ix[2].ix[0].ix[10],
-        3.8109046420620353e-12)
-
-def test_lte_partition_function(partition_function, levels):
-    assert np.allclose(partition_function.ix[2].ix[0], 1.0)
-    assert np.allclose(partition_function.ix[2].ix[1], 2.0)
-    assert np.allclose(partition_function.ix[2].ix[2], 1.0)
+from tardis import __path__ as path
+from tardis.io.util import yaml_load_config_file
+from tardis.simulation import Simulation
+from tardis.io.config_reader import Configuration
+from tardis.plasma.base import *
+
+@pytest.fixture(scope='module')
+def partition_compare_data_fname():
+    fname = 'partition_compare_data.h5'
+    return os.path.join(path[0], 'plasma', 'tests', 'data', fname)
+
+@pytest.fixture()
+def partition_compare_data(partition_compare_data_fname):
+    return h5py.File(partition_compare_data_fname, 'r')
+
+@pytest.mark.skipif(not pytest.config.getvalue("partition-test"),
+                    reason="partition tests are not included in this run")
```

Comment:
Yes, I have understood it now but the problem is to running it on the command line.